<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>120</y>
      <width>1041</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1041</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Filtro</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="filtro"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="tipo_filtro"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Topologia</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Tipo</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="topologia"/>
     </item>
    </layout>
   </widget>
   <widget class="QCustomPlot" name="magnitude" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>701</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="fase" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>420</y>
      <width>701</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="circuito">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="fase_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>701</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fase</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>701</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Magnitude</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="calc">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="freq_layout">
     <item>
      <widget class="QRadioButton" name="f_radiobutton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fc">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>FrequÃªncia de corte</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="freq_valor">
       <property name="maximum">
        <double>999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mult_fc"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="capacitor_layout">
     <item>
      <widget class="QRadioButton" name="cap_radiobutton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="comp">
       <property name="text">
        <string>Capacitor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="comp_valor">
       <property name="maximum">
        <double>999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mult"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="indutot_layout">
     <item>
      <widget class="QRadioButton" name="ind_radiobutton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="comp_2">
       <property name="text">
        <string>Indutor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="comp2_valor">
       <property name="maximum">
        <double>999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mult_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="largura_banda_layuot">
     <item>
      <widget class="QRadioButton" name="larg_banda_radiobutton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="larg_banda">
       <property name="text">
        <string>Largura de banda</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="larg_banda_valor">
       <property name="maximum">
        <double>999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="larg_banda_mult"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="res">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="calc_res">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="rlc_label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rlc_valor">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cap_radiobutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>comp</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cap_radiobutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>comp_valor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cap_radiobutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>mult</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ind_radiobutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>comp_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ind_radiobutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>comp2_valor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ind_radiobutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>mult_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>f_radiobutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>freq_valor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>f_radiobutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>mult_fc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>f_radiobutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>fc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>larg_banda_radiobutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>larg_banda</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>larg_banda_radiobutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>larg_banda_valor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>larg_banda_radiobutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>larg_banda_mult</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
